lua/snacks/words.lua

```diff
@@ -99,6 +100,13 @@ function M.is_enabled(opts)
     end
   end
 
+  if not vim.tbl_isempty(config.exclude_ft) then
+    local ft = vim.bo[vim.api.nvim_get_current_buf()].filetype
+    if vim.tbl_contains(config.exclude_ft, ft) then
+      return false
+    end
+  end
+
   local buf = opts.buf or vim.api.nvim_get_current_buf()
```

Comment:
Should you use the buf variable declared here?

Reply:
I didn't think `opts.buf` would ever be used, because we only cared about the current buffer. Either way I changed the implementation according to Folke's suggestion.